Jenkinsfile

```diff
@@ -24,6 +28,27 @@ pipeline {
                 withCredentials([usernamePassword(credentialsId: params.REGISTRY_URL, passwordVariable: 'TOKEN', usernameVariable: 'USERNAME')]) {
                     sh('. $WORKSPACE/bin/miniconda3/etc/profile.d/conda.sh; conda activate candig; echo ${TOKEN} | docker login ${REGISTRY_URL} -u ${USERNAME} --password-stdin; make docker-push OVERRIDE_REGISTRY=${REGISTRY_URL}')
                 }
+        stage('Build htsgetapp_server') {
+            steps {
+                sh '''
+                  cd ./lib/htsget-server/htsget_app
+                  ls -la
```

Comment:
Minor: Is this listing a requirement?

Reply:
yes Aman. I created a bash script and wanted the script to go in, check the folder and then run and execute jenkins pipeline script.